keyboards/splitkb/kyria/keymaps/lw/config.h

```diff
@@ -0,0 +1,34 @@
+/* Copyright 2022 Lucas Waelti (@LucasWaelti)
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#ifdef RGBLIGHT_ENABLE
+#    define RGBLIGHT_ANIMATIONS
```

Comment:
Could you explicitly define which animations are enabled? 
```suggestion
#define RGBLIGHT_EFFECT_BREATHING
#define RGBLIGHT_EFFECT_RAINBOW_MOOD
#define RGBLIGHT_EFFECT_RAINBOW_SWIRL
#define RGBLIGHT_EFFECT_SNAKE
#define RGBLIGHT_EFFECT_KNIGHT
#define RGBLIGHT_EFFECT_CHRISTMAS
#define RGBLIGHT_EFFECT_STATIC_GRADIENT
#define RGBLIGHT_EFFECT_RGB_TEST
#define RGBLIGHT_EFFECT_ALTERNATING
#define RGBLIGHT_EFFECT_TWINKLE
```

Reply:
Since I have not installed LEDs on my keyboard, I left these defines untouched (they are the same as the default keymap). LEDs are anyway disabled in `rule.mk` (`RGBLIGHT_ENABLE = no`). 

Do you want me to specify an animation anyway? If so, let me know which one you would like to see! I haven't played with these options and am not familiar with them.

Reply:
If you don't have LEDs on the board, it may be best to just remove this whole block (the rgblight stuff).

Reply:
Ok, I removed all RGB related defines in [this commit](https://github.com/qmk/qmk_firmware/pull/15729/commits/37b5c684aa810966f890749b810936b698d2c7f6).